spec/mailers/previews/startup_mailer_preview.rb

```diff
@@ -1,4 +1,40 @@
 class StartupMailerPreview < ActionMailer::Preview
+  def feedback_as_email_with_grade
```

Comment:
While testing the preview observed this:

![image](https://user-images.githubusercontent.com/24313597/221609554-6d16a563-7a07-4181-b51a-f0e15d4c9a44.png)

It says _Here is some feedback from XYZ on your recent _submission_ and immediately shows grading info and then the feedback. Let's have a clearly separate content for grading information. Maybe start off the email with 

_Your submission was awarded the following grades by XYZ_ 

-display grades-

_Here is some feedback on the submission_

Reply:
![image](https://user-images.githubusercontent.com/53794102/221936286-d5669694-f55a-4a0b-8927-7dbebed43244.png)

Reply:
For both grade and feedback the margin or tabbing is a bit excessive. We should either have a small margin or no margin.

Reply:
![image](https://user-images.githubusercontent.com/53794102/222429076-56079c7a-c953-42c8-b9f4-5c851ea1372b.png)

Fixed it with the help of @vinutv